graal-sdk/build.gradle

```diff
@@ -26,16 +22,31 @@ task buildGraalSDK(type: Exec) {
     }
 }
 
-////////////////////////////////////////////////////////////////////////////////////////////////////
+task sourcesJar(type: Jar) {
```

Comment:
Are these two tasks needed? Isn't it defined by default?

Reply:
Yes we can remove them and the (empty) jars appear to be created anyway. Removing them